apache-tomcat8/default.toml

```diff
@@ -0,0 +1,36 @@
+[server]
+port = "8080"
+shutdown-port = "8005"
+redirect-port = "8443"
+
+# Configuration for the AJP connector
+#enable-ajp-connector = "true"   # comment out line to disable AJP port
+ajp-port = "8009"
+
+# Tomcat utilizes SecureRandom to provide random values for session ids.  When starting Tomcat under
+# a newly created Habitat supervisor, Docker Container, etc., there is a long (> 50 seconds) delay
+# where SecureRandom gathers enough entropy to return random values.  To avoid this, a non-blocking
+# PRNG is passed to Tomcat for faster start up times.
+catalina-opts = "-Djava.security.egd=file:/dev/./urandom"
+
+[userdbrealm]
+roles = ["sysadmin", "deployer", "auditor"]
+
+[[userdbrealm.users]]
+username = "admin"
+password = "password"
+roles = "manager-gui,manager-script,manager-jmx,manager-status" 
+
+[[userdbrealm.users]]
+username = "bmeyer"
```

Comment:
Should your username be here, or something like `example-user` instead? Might be confusing to people as to why I have to have this `bmeyer` person here.